projects/demo/package.json

```diff
@@ -13,7 +13,7 @@
     "typecheck:watch": "tsc -b --watch",
     "copy-assets": "cp -R ./src/assets ./dist",
     "build": "yarn clean && yarn copy-assets && yarn typecheck && webpack",
-    "dev": "yarn clean && yarn copy-assets && concurrently \"yarn typecheck:watch\" \"webpack serve --config webpack.config.js\"",
+    "dev": "yarn clean && yarn copy-assets && concurrently \"webpack serve --config webpack.config.js\" \"yarn typecheck:watch\"",
```

Comment:
This is the kind of change that shouldn't be in this PR

Reply:
That was a stray change - will revert. Thank you for catching it